bleachbit/Special.py

```diff
@@ -42,10 +42,10 @@ def __shred_sqlite_char_columns(table, cols=None, where=""):
     cmd = ""
     if cols and options.get('shred'):
         cmd += "update %s set %s %s;" % \
-            (table, ",".join(["%s = randomblob(length(%s))" % (col, col)
```

Comment:
I am not sure what is the ask here.i have already revert back to zeroblob and randomblob.

as I said in one of previous post , firefox has already remove the unique contrain on url and we dont see the sql error anymore both on mac and windows.

thank you :)


Sent via the Samsung Galaxy Note® 3, an AT&T 4G LTE smartphone

<div>-------- Original message --------</div><div>From: Andrew Ziem <notifications@github.com> </div><div>Date:03/09/2016  11:57 PM  (GMT-05:00) </div><div>To: az0/bleachbit <bleachbit@noreply.github.com> </div><div>Cc: brahmastra2016 <Brahmastra2016@gmail.com> </div><div>Subject: Re: [bleachbit] Mac OSx - Update  (#123) </div><div>
</div>In bleachbit/Special.py:

> @@ -42,10 +42,10 @@ def __shred_sqlite_char_columns(table, cols=None, where=""):
>      cmd = ""
>      if cols and options.get('shred'):
>          cmd += "update %s set %s %s;" % \
> -            (table, ",".join(["%s = randomblob(length(%s))" % (col, col)
This is not related to Mac OS X, and I would like to fix it a different way, please.

—
Reply to this email directly or view it on GitHub.